isa/macros/scalar/test_macros.h

```diff
@@ -12,9 +12,9 @@
 
 #define TEST_CASE( testnum, testreg, correctval, code... ) \
 test_ ## testnum: \
+    li  TESTNUM, testnum; \
```

Comment:
I'm glad this change only ended up affecting a handful of tests; I would've expected it to have greater blast radius.